docs/research.md

```diff
@@ -0,0 +1,36 @@
+<h1> How has research been involved in informing the weather.gov 2.0 design? </h1>
+
+* Understanding the challenges that both internal and external users experience through open-ended “generative” research 
+* Understanding the ins and outs of the existing systems through onsite visit 
+* Getting quick feedback on in-progress designs and prototypes from folks that will use the website
+* Designing and implementing internal and external surveys to get continuous feedback 
+
+<h2> What research has the team done to date? </h2>
+<p>Last updated: August 2024 </p>
+
+<h3> Generative research </h3>
+
+| Name  | When  | Goals  | Insights  |
+| -------- | ------- | -------- | ------- |
+| Path Analysis | April 2023 | Initial research with stakeholders, partners and forecasters across NWS to understand the problem space | Users experience challenges finding and understanding content due to the organization of the website which is based on the NWS org structure. Significant inconsistencies make data hard to interpret, areas do not map to areas that decision makers use. |
+| WFO onsite - Sterling (DC) | July 2023 | Onsite tour + observations at Sterling WFO office (DC area). Includes in-depth notes on internal tools and systems (AWIPs, etc) | Understanding of current forecaster workflow and tools in creating forecasts and various types of content  |
+| Forecaster research | Sept 2023 | Validate our understanding of the forecaster experience before, during and after a weather hazard | Began to identify some common needs for a CMS experience |
+| WFO onsite - Nashville | Nov 2023 | Better understand partner experiences before, during and after a weather hazard | Identified partner painpoints during a hazard - post-event support, mapping and radar, etc |
+| R6 - Content Strategy testing | June 2024 | Understand what supplementary information users need beyond forecasts, and how they want to find and use them | Hypothesis of how to organize content, by locations that are relevant to decision maker areas.  |
+
+
+<h3> General public usability research </h3>
+
+| Name  | When  | Interest  | Usability| Relevance| Comprehension|
+| -------- | ------- | -------- | ------- |------- | -------- |
+| R1 - Usability (public) | Dec 2024 | ------- |------- | -------- |
+| R2 - Usability (public) | Jan 2024 |  Most participants said they would try out weather.gov alongside other apps and website | Some participants had some issues finding what they were looking for| Additional details required | Issues interpreting metrics |
```

Comment:
```suggestion
| R2 - Usability (public) | Jan 2024 |  🟢 Most participants said they would try out weather.gov alongside other apps and website | 🟡 Some participants had some issues finding what they were looking for | 🟡 Additional details required | 🟡 Issues interpreting metrics |
```

Reply:
I would love maintaining visual indicators in this table if possible.

- 🟢 
- 🟡 
- 🔴 
- ⚪  (for open question/uncertain use cases?)